src/com/lilithsthrone/game/character/race/Subspecies.java

```diff
@@ -3180,8 +3180,8 @@ private String getTaurEnding() {
 	}
 	
 	/**
-	 * @param The character whose subspecies's name is to be returned. Can pass in null.
-	 * @return The singular name of this character's subspecies.
+	 * @param   The character whose subspecies's name is to be returned. Can pass in null.
+	 * @return  The singular name of this character's subspecies.
```

Comment:
Non-changes like this tend to clutter things up. While these here are probably fine, it's a good idea to try to avoid them in the future.

Reply:
Ah, yes, that. IntelliJ Idea would annoy me a throw an error each time because it couldn't recognize "The" so I put some spaces